tests/snapshots/readback_hvm__bad_net.bend.snap

```diff
@@ -3,6 +3,7 @@ source: tests/golden_tests.rs
 input_file: tests/golden_tests/readback_hvm/bad_net.bend
 ---
 [4m[1m[33mWarnings:[0m
+[1mDuring execution :[0m
```

Comment:
Looking at all the errors as a whole, I actually think the `During execution` is not actually adding anything. In all cases we have here we could just remove this line and the comprehension of the message wouldn't be affected.

Also, for most things it should actually be a "during compilation", right?